lib/data-types.js

```diff
@@ -38,12 +39,20 @@ var ABSTRACT = function(options) {
 
 };
 
+ABSTRACT.prototype.dialectTypes = '';
+
 ABSTRACT.prototype.toString = function() {
   return this.toSql();
 };
 ABSTRACT.prototype.toSql = function() {
   return this.key;
 };
+ABSTRACT.prototype.warn = function(text) {
+  if (!warnings[text]) {
+    warnings[text] = true;
+    console.warn('>> WARNING:', text, '\n>> Check:', this.dialectTypes);
```

Comment:
Hm that would be nice indeed... It seems like depd [includes 'deprecated' hardcoded](https://github.com/dougwilson/nodejs-depd/blob/master/index.js#L100). I could include the colours without difficulties though :D